test/IDE/print_stdlib.swift

```diff
@@ -2,6 +2,9 @@
 //
 // REQUIRES: nonexecutable_test
 // REQUIRES: swift_swift_parser
+// FIXME: Swift parser is not enabled on Linux CI yet.
+// REQUIRES: OS=macosx
```

Comment:
I assume you ran into whatever I ran into in https://github.com/apple/swift/pull/64089 that caused me not to merge it, though I don't remember what it was 😅

Reply:
YEAH :(